src/components/report/ReportHTMLDownload.svelte

```diff
@@ -34,84 +35,173 @@
 
 <div hidden use:cleanUp bind:this={htmlDownloadTemplate}>
   <style>
```

Comment:
We had wanted to have an external style sheet, but if we did that the html document couldn't be a stand alone document (MHTML), right? 

Just trying to figure out why we're replacing so much CSS here.

Reply:
Ya I created #11 to solve that but we might pivot to the CLI first and then come back to the editor.

Reply:
Even in the CLI using inline CSS to make it easier for the maintainer to update just one file. The reason for the editor including CSS is because I couldn't figure out earlier how to the style changes and I guess the zip file would allow additional of external CSS files. Edited the description of the issue.